lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb

```diff
@@ -240,7 +240,7 @@ def self.download_from_glotpress(res_dir:, glotpress_project_url:, glotpress_fil
           attributes_to_copy = %w[formatted] # Attributes that we want to replicate into translated `string.xml` files
           orig_file = File.join(res_dir, 'values', 'strings.xml')
           orig_xml = File.open(orig_file) { |f| Nokogiri::XML(f, nil, Encoding::UTF_8.to_s) }
-          orig_attributes = orig_xml.xpath('//string').map { |tag| [tag['name'], tag.attributes.select { |k, _| attributes_to_copy.include?(k) }] }.to_h
+          orig_attributes = orig_xml.xpath('//string').to_h { |tag| [tag['name'], tag.attributes.select { |k, _| attributes_to_copy.include?(k) }] }
```

Comment:
Same question here as https://github.com/wordpress-mobile/release-toolkit/pull/464/files#r1174292121

Reply:
Same. I'm pretty sure the return value of `xpath` is an `Enumerable`, so 👍